modules/social_features/social_event/modules/social_event_managers/social_event_managers.module

```diff
@@ -356,13 +357,25 @@ function social_event_managers_form_node_event_form_alter(&$form, FormStateInter
 /**
  * Implements hook_activity_recipient_organizer_alter().
  */
-function social_event_managers_activity_recipient_organizer_alter(array &$recipients, Node $event) {
+function social_event_managers_activity_recipient_organizer_alter(array &$recipients, Node $event, $data) {
+  $receiver = '';
   $organizers = $event->get('field_event_managers')->getValue();
 
+  if ($data['target_type'] === 'event_enrollment' && !empty($data['target_id'])) {
+    $enrollment = EventEnrollment::load($data['target_id']);
+    $receiver = $enrollment->getAccount();
+  }
+
   // If there are more organizers we want them to receive a notification too
   // so we add them to the array of recipients.
   if (!empty($organizers)) {
     foreach ($organizers as $organizer) {
+      // We don't want Organizers to receive activity_on_events_im_organizing.
+      // He will already receive it as part of a different context.
+      if (!empty($receiver) && $organizer['target_id'] === $receiver) {
```

Comment:
@ronaldtebrake of which context? And should this be solved this way? Why even create the activity in the first place then 🤔 

I'm just wondering, because it could be that user disabled notifications of that other context, but want to receive this one.